### ISSUE TITLE
Add Preferred Pronouns

### ISSUE THREAD

**Issue by nosbig** [monicahq/monica#3513]
As gender identify becomes more fluid, it might be nice to be able to track preferred pronouns for people who choose something non-default.  It could be tracked as notes, but it might also be nice to have a dedicated field for this, perhaps using the current "gender" tracking for the defaults for a specific pronoun.

**Comment by djaiss** [comment]
Oh, you mean "he/she" etc... ?

**Comment by nosbig** [comment]
Yes.  I have seen people that prefer he/him, she/her, they/them, and zhe/zher.

**Comment by vzhz** [comment]
Hello @djaiss, I see you currently have a gender field that allows male, female, or "other" pronouns. Would you be open to the addition of "they/them" and "zhe/zher"?
If so, I'd be happy to work on this change.

**Comment by asbiin** [comment]
@vzhz it's possible to add it to your account. I don't think it's relevant to add these value to all accounts.

**Comment by mnemonica** [comment]
"Prounouns" should probably be a freeform text field instead of an enum: this would allow common combinations like "she/they" "ze/they" etc without having to worry about building every possible combination into a combobox.

I don't think pronouns should necessarily be tied to the gender field as vzhz suggests, because they don't map 1:1 in real life: there are many people who are nonbinary but use pronouns like "she", "he/they", "they/zir", etc.

Finally, if we implement this, we should be careful to label the field simply "pronouns" instead of "preferred pronouns", because [saying “preferred” makes it seem like using someone’s pronouns is optional when, in reality, using a person’s pronouns is the most basic need they have to feel safe and to exist in public spaces](https://lgbtresourcecenter.uccs.edu/pronouns-101).

**Comment by lazerwalker** [comment]
+1 to @mnemonica. A freeform text field for "pronouns" (_not_ "preferred pronouns") would be an incredibly welcomed addition, and go a long way towards the app's handling of gender.

**Comment by puppycodes** [comment]
> Thanks for your input, all. So basically, here, the need is to add a pronoun field.
> 
> And this pronoun field ideally, would be a free text field.

Yes that would be wonderful! Thanks so much.

**Comment by puppycodes** [comment]
Just wanted to uplift this thread once again

**Comment by Kotarosz727** [comment]
input pronoun is optional?

**Comment by nosbig** [comment]
I feel as if the pronoun field should be optional, as not all people care
to use pronouns, but it is good for it to be present for those who wish to
track that or those who have a preference outside historical "societal
norms".

Thanks,
Rob

**Comment by Kotarosz727** [comment]
Thanks! I will take care of this task.

**Comment by Kotarosz727** [comment]
@djaiss san
I will add pronoun input field beneath Gender field.(/people/add, people/edit ,/relationship/add)
Should you have any concern, kindly let me know. 

Thank you.
<img width="708" alt="image" src="https://user-images.githubusercontent.com/50207951/152793371-ad48249d-256e-4494-8dd7-63d7fd8423e3.png">

**Comment by lazerwalker** [comment]
This is great! In English, that field should probably be labeled "Pronouns" (plural) instead of "Pronoun".

**Comment by djaiss** [comment]
The new version (codename Chandler) has this feature built-in.

<img width="511" alt="image" src="https://user-images.githubusercontent.com/61099/207741963-4dedba43-0af6-44fd-bf98-d7d0cde248ad.png">

Also, pronouns are fully customizable (account wide)

<img width="750" alt="image" src="https://user-images.githubusercontent.com/61099/207742007-36178675-13ac-4f15-a129-5ebdc6d41b81.png">

**Comment by puppycodes** [comment]
> The new version (codename Chandler) has this feature built-in.
> 
> <img alt="image" width="511" src="https://user-images.githubusercontent.com/61099/207741963-4dedba43-0af6-44fd-bf98-d7d0cde248ad.png">
> 
> Also, pronouns are fully customizable (account wide)
> 
> <img alt="image" width="750" src="https://user-images.githubusercontent.com/61099/207742007-36178675-13ac-4f15-a129-5ebdc6d41b81.png">

tyyyyyyy I finallly want to use this again :) 🥹 🔥

**Comment by djaiss** [comment]
New version is coming soon (https://github.com/monicahq/chandler)

**Comment by pablo-moraes** [comment]
Shouldn't this issue to be closed?
Considering that Chandler already has this feature.
